Splash/Splash.xcworkspace/xcuserdata/shreyavinjamuri.xcuserdatad/xcdebugger/Breakpoints_v2.xcbkptlist

```diff
@@ -685,8 +685,8 @@
             filePath = "Splash/Networking/SpotifyAPIManager.m"
```

Comment:
This file should not be part of commit. I think we need to add this to ignore file if not already. 

If there is no ignore file, just look up git ignore file and how to add it in the repo.